cmake/dependencies.cmake

```diff
@@ -94,22 +94,27 @@ function(fetch_dependencies)
         set(FBS_GENERATED_HEADER ${FBS_GENERATED_HEADER} PARENT_SCOPE)
     endfunction()
 
-    ############################################################################################################################
+    ###################################################################################################################
     # libuv (for process management)
-    ############################################################################################################################
+    ###################################################################################################################
     CPMAddPackage(NAME libuv GITHUB_REPOSITORY libuv/libuv GIT_TAG v1.48.0 OPTIONS "LIBUV_BUILD_TESTS OFF")
 
-    ############################################################################################################################
+    ###################################################################################################################
     # fmt : https://github.com/fmtlib/fmt
-    ############################################################################################################################
+    ###################################################################################################################
 
     CPMAddPackage(NAME fmt GITHUB_REPOSITORY fmtlib/fmt GIT_TAG 11.0.1)
 
-    ############################################################################################################################
+    ###################################################################################################################
     # nanobench (for uBenchmarking)
-    ############################################################################################################################
+    ###################################################################################################################
     if(MASTER_PROJECT)
         CPMAddPackage(NAME nanobench GITHUB_REPOSITORY martinus/nanobench GIT_TAG v4.3.11)
     endif()
+
+    ####################################################################################################################
+    # spdlog
+    ####################################################################################################################
+    CPMAddPackage(NAME spdlog GITHUB_REPOSITORY gabime/spdlog GIT_TAG v1.14.1 VERSION v1.14.1)
```

Comment:
Is this used somewhere else in our repos, or are we the first to introduce the usage?

Reply:
We're the first as far as I know